Gemfile

```diff
@@ -12,7 +12,7 @@ group :development do
   gem 'berkshelf', '~> 3.0'
 
   # Use Test Kitchen with Vagrant for converging the build environment
-  gem 'test-kitchen',    '~> 1.4.0.rc.1'
-  gem 'kitchen-vagrant', '~> 0.17.0.rc.1'
+  gem 'test-kitchen',    '~> 1.4.0'
```

Comment:
It does - the existing Gemfile.lock already pulls in a more recent test kitchen.  Since I needed to run bundle update to pull in updated deps for omnibus as is, I figured I'd just clean up the "rc" stuff because I found it ugly :)